__tests__/app.test.js

```diff
@@ -91,3 +90,67 @@ describe('/api/articles/:article_id', () => {
             });
     });
 });
+
+describe('/api/articles', () => {
+    test('GET - status: 200 - returns the correct information', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.articles.length).toBe(12);
+            });
+    });
+    test('GET - status: 200 - all required columns are returned', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                response.body.articles.forEach((article) => {
+                    expect(typeof article.author).toBe('string');
+                    expect(typeof article.title).toBe('string');
+                    expect(typeof article.article_id).toBe('number');
+                    expect(typeof article.topic).toBe('string');
+                    expect(typeof article.created_at).toBe('string');
+                    expect(typeof article.votes).toBe('number');
+                    expect(typeof article.article_img_url).toBe('string');
+                    expect(typeof article.comment_count).toBe('number');
+                });
+            });
+    });
+    test('GET - status: 200 - sort by created_at', () => {
+        return request(app)
+            .get('/api/articles?sort_by=created_at')
```

Comment:
Currently we'd only like to see _default_ sortBy, meaning you can still make this assertion but with /api/articles. It looks like you've assigned default variables so this should still pass anyway.